src/node/cli.ts

```diff
@@ -157,6 +158,10 @@ export const options: Options<Required<UserProvidedArgs>> = {
       "Disable update check. Without this flag, code-server checks every 6 hours against the latest github release and \n" +
       "then notifies you once every week that a new release is available.",
   },
+  "disable-file-downloads": {
+    type: "boolean",
+    description: "Disable file downloads from Code. Defaults to true.",
```

Comment:
It looks like the default is to not disable downloads, right?  (Which I think is the right move.)

```suggestion
    description: "Disable file downloads from Code.",
```